bcda/auth/client/ssas_test.go

```diff
@@ -144,6 +144,7 @@ func (s *SSASClientTestSuite) TestResetCredentials() {
 	}
 
 	resp, err := client.ResetCredentials("1")
+	fmt.Println(string(resp))
```

Comment:
This may have been left in unintentionally

Reply:
removed :]